katdal/chunkstore_npy.py

```diff
@@ -0,0 +1,73 @@
+################################################################################
```

Comment:
Because, er, `DictChunkStore` does not depend on `numpy`, but `NpyFileChunkStore` does... The former is meant to be the most generic useful chunk store I could think of, while the latter is a special workaround for @sratcliffe to cache chunks on site. And I grew tired of yet another mini-file where the largest contribution is the license boilerplate. :-)

Reply:
`DictChunkStore` might not need to explicitly import `numpy`, but is it realistically usable without numpy installed, given that the arrays themselves are either numpy arrays? (or numpy-like, but h5py and I'd guess any other package that provides numpy-like arrays will depend on numpy anyway).

Reply:
Moved `DictChunkStore` to its own file as the base class got too big :-)